stdlib/test/builtin/test_object.mojo

```diff
@@ -10,11 +10,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ===----------------------------------------------------------------------=== #
-# RUN: %mojo %s | FileCheck %s
+# RUN: %mojo-no-debug %s
```

Comment:
```suggestion
# RUN: %mojo %s
```

We're moving away from using `%mojo-no-debug` in the rest of the standard library tests, so it would be great to update this here as well.